client/my-sites/stats/all-time/index.jsx

```diff
@@ -94,22 +94,22 @@ module.exports = React.createClass( {
 					<li className="stats-all-time__list-item stats-all-time__item-posts">
 						<Gridicon icon="posts" size={ 18 } />
 						<span className="stats-all-time__label">{ this.translate( 'Posts' ) }</span>
-						<span className={ valueClass }>{ this.ensureValue( this.props.allTimeList.response.posts ) }</span>
+						<span className={ classNames( valueClass, { 'is-low': this.props.allTimeList.response.posts === 0 } ) }>{ this.ensureValue( this.props.allTimeList.response.posts ) }</span>
```

Comment:
@jancavan This is working great as-is, but I think we might want to do something similar to the `ensureValue` function.  That would allow us to add a check to make sure all of these values exist before seeing if they are equal to 0.  So we don't accidentally try to compare `undefined` and throw an error.

Thinking something like:

```
isLow: function( value ) {
    return ! value || 0 === value
}
```

then in your calls to `classNames` you could have it look like `{ 'is-low': this.isLow( this.props.allTime... } }`